scenarios/kubernetes_kops_aws.py

```diff
@@ -165,6 +165,8 @@ def main(args):
                (args.cluster, zones, args.state, args.nodes, aws_ssh))
     if args.image:
         e2e_opt += ' --kops-image=%s' % args.image
+    if args.kops_args:
+        e2e_opt += ' --kops-args=\"%s\"' % args.kops_args
```

Comment:
I suspect the person who added the previous flags was simply unaware of the ability to these as cmd line arguments